src/View/Helper/FormHelper.php

```diff
@@ -276,4 +234,124 @@ protected function _parseOptions($fieldName, $options)
         }
         return $options;
     }
+
+    /**
+     * Form alignement detector/switcher.
+     *
+     * @param array $options Options.
+     * @return array Modified options.
+     */
+    protected function _formAlignement($options)
+    {
+        if (!$options['align']) {
+            $options['align'] = $this->_detectFormAlignement($options);
+        }
+
+        if (is_array($options['align'])) {
+            $this->_grid = $options['align'];
+            $options['align'] = 'horizontal';
+        } elseif ($options['align'] === 'horizontal') {
+            $this->_grid = $this->config('grid');
+        }
+
+        if (!in_array($options['align'], ['default', 'horizontal', 'inline'])) {
+            throw new InvalidArgumentException('Invalid `align` option value.');
+        }
+
+         $this->_align = $options['align'];
+
+         unset($options['align']);
+
+        if ($this->_align === 'default') {
+            $options['templates'] += [
+               'checkboxContainer' => '<div class="checkbox">{{content}}{{help}}</div>',
```

Comment:
Sorry I misread the code. 

Still perhaps for easier management and better readability it might be better if these extra template sets were keep in a var instead of having them inline in the method.